torch/_inductor/fx_passes/mkldnn_fusion.py

```diff
@@ -1006,7 +1006,6 @@ def linear(match, *args, **kwargs):
                     if free_symbols(batch_size)
                     else batch_size,
                 )
-                packed_weight_inputs = (transpose_weight_node, batch_size)
```

Comment:
is this fix covered by some ut?

Reply:
our test case has a dynamic shape case, and it can work even has this line, but I meet a model case, this line will remove the symbolic shape size to a real number and will recompile if the input size changes, I tried to break down a smaller case, but failed.